server/views/pages/goal/review/index.njk

```diff
@@ -18,36 +17,46 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">Review {{ prisonerSummary.firstName | capitalize }} {{ prisonerSummary.lastName | capitalize }}'s goals</h1>
-    </div>
-  </div>
-
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
 
-        {{ govukTable({
-          caption: 'Goal 1: ' + data.title + '',
-          captionClasses: "govuk-table__caption--m",
-          firstCellIsHeader: false,
-          head: [
-            { text: "Steps" },
-            { text: "Target date" }
-          ],
-          rows: data.steps | stepsTableRowFilter,
-          attributes: { id: 'steps-table' }
+      <div class="govuk-summary-card">
```

Comment:
This block is where I've used vanilla HTML instead of a Nunjucks macro. The HTML outputted is the exact same as the Nunjucks macro, however it means that can handle all the view logic in this HTML file opposed to having another complex file to populate the Nunjucks macro fields.

Reply:
I also dont feel best qualified to say which is better. I guess its on a case by case basis, and probably based around the amount of code we'd have to write either setting variables in the view (😬 ) and/or filters and such like

What you've done looks fine though, and I can see why you are proposing this approach.

My only caution here would be that we dont take this approach as our defacto / go-to approach. I think we should use the govuk macros where ever possible/practical; and only look to do what we've done here when there is a clear benefit / reason for doing so 👍

Reply:
Yep completely agree with that!